frontend/pages/hosts/ManageHostsPage/helpers.ts

```diff
@@ -43,11 +105,70 @@ export const getNextLocationPath = (
   }
 
   const nextLocation = union(
-    trim(pathPrefix, "/").split("/"),
+    trim(pathPrefixFinal, "/").split("/"),
     routeString.split("/")
   ).join("/");
 
   return queryString ? `/${nextLocation}?${queryString}` : `/${nextLocation}`;
 };
 
+const getSortedTeamOptions = memoize((teams: ITeam[]) =>
+  teams
+    .map((team) => {
+      return {
+        disabled: false,
+        label: team.name,
+        value: team.id,
+      };
+    })
+    .sort((a, b) => sortUtils.caseInsensitiveAsc(b.label, a.label))
```

Comment:
```suggestion
    .sort((a, b) => sortUtils.caseInsensitiveAsc(a.label, b.label))
```

Small change to track my fix to the semantics of sort utilities in #2105